### PR TITLE
Feature: go to admin

### PR DESCRIPTION
**Pull request by Zorin95670**
Add button to go to administration view and share token between leto-modelizer and leto-modelizer-admin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,24 +165,38 @@ When you have installed all the desired plugins, please run this commands `npm r
 
 We are using [Leto-Modelizer-Api](https://github.com/ditrit/leto-modelizer-api) authentication (as a backend). To configure it, you must fill the configuration inside `global.config.json` root file.
 
+We have administration view [Leto-Modelizer-Admin](https://github.com/ditrit/leto-modelizer-admin), for user who have `admin` role. To manage users, groups and roles.
```
Comment (k-y0u) [inline README.md:168]: for users who have `admin` role (to manage users, groups and roles).
OR
for users who have `admin` role, to manage users, groups and roles.

File: README.md [hunk 2]
```diff
@@ -165,24 +165,38 @@ When you have installed all the desired plugins, please run this commands `npm r
...
-        "appId": "leto-modelizer-api-dev"
+        "appId": "leto-modelizer-api-dev",
+        "adminUrl": "http://localhost:9000"
     }
 }
 ```
 
 All the fields inside `backend` are mandatory.
 
+To have authentication setup, `backend.url` and `backend.appId` are mandatory.
```
Comment (k-y0u) [inline README.md:182]: Having authentication setup, `backend.url` and `backend.appId` is mandatory.

File: src/components/menu/ModelizerSettingsMenu.vue [hunk 3]
```diff
@@ -51,6 +51,23 @@
           </q-item-section>
         </q-item>
       </template>
+      <q-item
+        v-can:admin
+        clickable
+        class="settings-item"
+        data-cy="item_admin"
```
Comment (justinehell) [inline src/components/menu/ModelizerSettingsMenu.vue:58]: why not the same ?

File: README.md [hunk 4]
```diff
@@ -165,24 +165,38 @@ When you have installed all the desired plugins, please run this commands `npm r
...
+To have administration view, all the fields inside `backend` are mandatory.
```
Comment (k-y0u) [inline README.md:183]: Having administration view and all the fields inside `backend` is mandatory.

**Review by justinehell — CHANGES_REQUESTED** [review]

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -165,24 +165,38 @@ When you have installed all the desired plugins, please run this commands `npm r
...
 **_NOTE:_**: If the previous configuration is not present in the configuration file, Leto-Modelizer will be launched with the backend mode deactivated.
 **_NOTE:_**: For now, there is no UI associated to the backend, but the UI for the admin is coming soon !
 
+## How to setup the administration
+
+You need to follow the installation part of:
+- [Leto-Modelizer-Api](https://github.com/ditrit/leto-modelizer-api)
+- [Leto-Modelizer-Admin](https://github.com/ditrit/leto-modelizer-admin)
+
+
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply README.md:199]: done

File: README.md [hunk 6]
```diff
@@ -165,24 +165,38 @@ When you have installed all the desired plugins, please run this commands `npm r
...
+- `adminUrl`: the url of the administration application.
 
 **_NOTE:_**: If the previous configuration is not present in the configuration file, Leto-Modelizer will be launched with the backend mode deactivated.
 **_NOTE:_**: For now, there is no UI associated to the backend, but the UI for the admin is coming soon !
 
+## How to setup the administration
+
+You need to follow the installation part of:
+- [Leto-Modelizer-Api](https://github.com/ditrit/leto-modelizer-api)
+- [Leto-Modelizer-Admin](https://github.com/ditrit/leto-modelizer-admin)
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply README.md:197]: forget to remove this part, done

File: README.md [hunk 7]
```diff
@@ -165,24 +165,38 @@ When you have installed all the desired plugins, please run this commands `npm r
...
+- `url`: the url of the backend.
+- `appId`: the application ID of the backend.
+- `adminUrl`: the url of the administration application.
 
 **_NOTE:_**: If the previous configuration is not present in the configuration file, Leto-Modelizer will be launched with the backend mode deactivated.
 **_NOTE:_**: For now, there is no UI associated to the backend, but the UI for the admin is coming soon !
 
+## How to setup the administration
+
+You need to follow the installation part of:
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply README.md:195]: forget to remove this part, done

File: README.md (hunk 1 above)
Comment (Zorin95670) [reply README.md:168]: done

**Review by justinehell — APPROVED** [review]

**Review by justinehell — APPROVED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**